src/main/java/frc/robot/subsystems/Intake.java

```diff
@@ -0,0 +1,39 @@
+// Copyright (c) FIRST and other WPILib contributors.
+// Open Source Software; you can modify and/or share it under the terms of
+// the WPILib BSD license file in the root directory of this project.
+
+package frc.robot.subsystems;
+import frc.robot.Constants;
+import com.revrobotics.CANSparkMax;
+import com.revrobotics.CANSparkMaxLowLevel.MotorType;
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
+
+public class Intake extends SubsystemBase {
+
+  /** Creates a new Intake. */
+  private CANSparkMax intakeMotor = new CANSparkMax( Constants.INTAKE_MOTOR, MotorType.kBrushless);
+  public Intake() {
+
+  }
+
+  public void conePickup(double speed){
+    intakeMotor.set(speed * -1);
+  }
+
+  public void coneDrop(double speed){
+    intakeMotor.set(speed * 1);
+  }
+
+  public void cubeDrop(double speed){
+    intakeMotor.set(speed * -1);
+  }
+
+  public void cubePickup(double speed){
+    intakeMotor.set(speed * 1);
+  }
```

Comment:
Need to add a stop method to this subsystem to be called from the end of each command